doc/user/install.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Install"
+description: "Get started with Materialize"
+menu: "main"
+weight: 2
+---
+
+You can access Materialize through the `materialized` binary, which you can install on macOS and Linux.
+
+## macOS installation
+
+### Homebrew
+
+Assuming you've installed [Homebrew](https://brew.sh/):
+
+```shell
+brew install materialized
```

Comment:
```
brew install MaterializeInc/materialize/materialized
```

Reply:
Ah, I thought I pushed a change to this which I clearly didn't; is this valid/good?

```
brew tap MaterializeInc/homebrew-materialize
brew install materialized
```

Taken from https://github.com/MaterializeInc/homebrew-materialize/pull/1

Reply:
I think we should be encouraging the one-liner that I posted above! I’ll update the README in the other repo to read the same.